lib/demo_web/templates/user_session/new.html.eex

```diff
@@ -0,0 +1,27 @@
+<h1>Login</h1>
+
+<%= form_for @conn, Routes.user_session_path(@conn, :create), [as: :user], fn f -> %>
+  <%= if @error_message do %>
+    <div class="alert alert-danger">
+      <p><%= @error_message %></p>
+    </div>
+  <% end %>
+
+  <%= label f, :email %>
+  <%= text_input f, :email, required: true %>
+
+  <%= label f, :password %>
+  <%= password_input f, :password, required: true %>
+
+  <%= label f, :remember_me, "Keep me logged in for 60 days" %>
+  <%= checkbox f, :remember_me %>
+
+  <div>
+    <%= submit "Login" %>
```

Comment:
I realise this is totally nit-picking, so feel free to ignore me, but... "login" is a noun, "log in" is the verb. So to me action-oriented UI messages and function names should be "log in"/`log_in`. It's the sort of thing I notice but I know I'm a total pedant so :shrug:

Reply:
Jinx: https://github.com/aaronrenner/phx_gen_auth/issues/32

Reply:
Good catch @jonleighton! We actually fixed this in our app later but I forgot to back-port it here. :)